src/main/java/org/mskcc/oncokb/curation/config/security/SecurityConfigurationOAuth.java

```diff
@@ -1,114 +1,110 @@
-        // @formatter:on
+            .cors(withDefaults())
+            .csrf(
+                csrf ->
+                    csrf
+                        .csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse())
+                        .csrfTokenRequestHandler(new SpaCsrfTokenRequestHandler())
+            )
+            .addFilterAfter(new SpaWebFilter(), BasicAuthenticationFilter.class)
+            .addFilterAfter(new CookieCsrfFilter(), BasicAuthenticationFilter.class)
+            .headers(
+                headers ->
+                    headers
+                        .contentSecurityPolicy(csp -> csp.policyDirectives(jHipsterProperties.getSecurity().getContentSecurityPolicy()))
+                        .frameOptions(HeadersConfigurer.FrameOptionsConfig::sameOrigin)
+                        .referrerPolicy(
+                            referrer -> referrer.policy(ReferrerPolicyHeaderWriter.ReferrerPolicy.STRICT_ORIGIN_WHEN_CROSS_ORIGIN)
+                        )
+                        .permissionsPolicy(
+                            permissions ->
+                                permissions.policy(
+                                    "camera=(), fullscreen=(self), geolocation=(), gyroscope=(), magnetometer=(), microphone=(), midi=(), payment=(), sync-xhr=()"
+                                )
+                        )
+            )
+            .authorizeHttpRequests(
+                authz ->
+                    // prettier-ignore
+                authz
+                    .requestMatchers(mvc.pattern("/index.html"), mvc.pattern("/*.js"), mvc.pattern("/*.txt"), mvc.pattern("/*.json"), mvc.pattern("/*.map"), mvc.pattern("/*.css")).permitAll()
+                    .requestMatchers(mvc.pattern("/*.ico"), mvc.pattern("/*.png"), mvc.pattern("/*.svg"), mvc.pattern("/*.webapp")).permitAll()
+                    .requestMatchers(mvc.pattern("/app/**")).permitAll()
+                    .requestMatchers(mvc.pattern("/i18n/**")).permitAll()
+                    .requestMatchers(mvc.pattern("/content/**")).permitAll()
+                    .requestMatchers(mvc.pattern("/swagger-ui/**")).permitAll()
+                    .requestMatchers(mvc.pattern("/api/authenticate")).permitAll()
+                    .requestMatchers(mvc.pattern("/api/auth-info")).permitAll()
+                    .requestMatchers(mvc.pattern("/api/logout")).permitAll()
+                    .requestMatchers(mvc.pattern("/api/admin/**")).hasAuthority(AuthoritiesConstants.ADMIN)
+                    .requestMatchers(mvc.pattern("/api/audit/**")).hasAuthority(AuthoritiesConstants.ADMIN)
+                    .requestMatchers(mvc.pattern("/api/account/firebase-token")).hasAnyAuthority(AuthoritiesConstants.CURATOR)
+                    .requestMatchers(mvc.pattern("/websocket/**")).hasAnyAuthority(AuthoritiesConstants.CURATOR)
+                    .requestMatchers(mvc.pattern("/api/**")).hasAnyAuthority(AuthoritiesConstants.CURATOR, AuthoritiesConstants.ADMIN)
```

Comment:
@calvinlu3 should I replace CURATOR with USER in this file?

Reply:
I think it should probably include both CURATOR and USER because the curation page still needs to make some api calls. It doesn't matter right now because all CURATORS are also USERS, but may be an issue if someone is only a curator.

Reply:
Can anyone be CURATOR but not USER? I feel like USER should be a default role for everyone.

Reply:
I would need to verify if we give USER by default. If that's the case, then replacing would be fine